xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -257,8 +265,7 @@
         <param name="utf8Destination">The destination buffer to which the encoded form of <paramref name="utf8Source" /> will be written.</param>
         <param name="bytesConsumed">The number of bytes consumed from the <paramref name="utf8Source" /> buffer.</param>
         <param name="bytesWritten">The number of bytes written to the <paramref name="utf8Destination" /> buffer.</param>
-        <param name="isFinalBlock">
-          <see langword="true" /> if there is further source data that needs to be encoded; otherwise, <see langword="false" />.</param>
+        <param name="isFinalBlock"><see langword="true" /> if there is no further source data that needs to be encoded; otherwise, <see langword="false" />.</param>
```

Comment:
Can you add the following to match with param from [above](https://github.com/dotnet/dotnet-api-docs/pull/3151#discussion_r322477351).
```suggestion
        <param name="isFinalBlock"><see langword="true" /> to indicate there is no further source data that needs to be encoded; otherwise, <see langword="false" />.</param>
```